### PR TITLE
Merge RedAutoRewrite into Auto_FixPath

### PR DESCRIPTION
**Pull request by ghost**
- Created a tested version of the Auto that works for Red
- Math for AprilTags need to be adjusted for the actual event
- Might want to adjust the code where it interacts with the can a little bit, since it seems to not work as well.

Before issuing a pull request, please see the contributing page.

### CODE REVIEW

**Review by gbtsui — APPROVED** [review]
Looks good for now, I'll wait for Pablo's approval before merging

**Comment by ghost** [comment]
d

**Comment by gbtsui** [comment]
> d

???

**Pull request closed without merging**
